py3status/argparsers.py

```diff
@@ -60,6 +60,13 @@ def parse_cli_args():
     except subprocess.CalledProcessError:
         i3status_path = None
 
+    # get swaymsg or i3-msg
```

Comment:
obsolete comment because this is not the only difference between sway and i3... maybe just say "try to detect a good default for "wm" argument" ?